examples/simple/sagas.js

```diff
@@ -0,0 +1,28 @@
+import { parse } from 'query-string';
+import { getLocale, changeLocale } from 'react-admin';
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+import { LOCATION_CHANGE } from 'react-router-redux';
+
+const localeParam = 'locale';
+
+function* handleChangeLocaleAction({ payload: { search } }) {
```

Comment:
I'm not fan of that use case - we shouldn't encourage messing up with the url. Changing the locale should be a function exposed in the UI. Besides, to illustrate how a custom saga can change the messages, why don't you illustrate the use of cookies for the locale?